ethclient/gethclient/gethclient.go

```diff
@@ -176,7 +176,12 @@ func (ec *Client) GetNodeInfo(ctx context.Context) (*p2p.NodeInfo, error) {
 }
 
 // SubscribePendingTransactions subscribes to new pending transactions.
-func (ec *Client) SubscribePendingTransactions(ctx context.Context, ch chan<- common.Hash) (*rpc.ClientSubscription, error) {
+func (ec *Client) SubscribePendingTransactions(ctx context.Context, ch chan<- *types.Transaction) (*rpc.ClientSubscription, error) {
```

Comment:
As the gethclient is relatively new I find this breaking change tempting. It would "correct" the wording, as `SubscribePendingTransactions` never subscribed to transactions, but only to transaction hashes.

Reply:
@fjl what do you think?

Reply:
@fjl

Reply:
Sorry for not responding. I think we should not change the API, but this is just me being very strict about things like that. I agree the current naming is unfortunate.